Source/SIMPLib/Common/PhaseType.h

```diff
@@ -83,7 +83,7 @@ class SIMPLib_EXPORT PhaseType
    * @param types
    * @return
    */
-  static Types FromQVector(QVector<EnumType> &types);
+  static Types FromQVector(QVector<EnumType>& vec);
```

Comment:
types has more meaning than vec.  It's minor, but this probably shouldn't be changed.